client/src/index.js

```diff
@@ -1,12 +1,18 @@
 // PR comment
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react";
+import ReactDOM from "react-dom";
+import { BrowserRouter as Router, Route } from "react-router-dom";
 
-import './index.css';
-import MovieList from './Movies/MovieList';
-import Movie from './Movies/Movie'
+import "./index.css";
+import MovieList from "./Movies/MovieList";
+import Movie from "./Movies/Movie";
 
 ReactDOM.render(
-  <div>Application running, add your routing</div>,
-  document.getElementById('root')
+  <Router>
+    <div>
+      <Route exact path="/" component={MovieList} />
+      <Route path="/movies/:movieId" component={Movie} />
```

Comment:
I checked up on that interchangeability question you brought up yesterday. I notice that whichever `param` you declare here - in your case, `movieId` - will be the value set for the `params` property on the `match` object passed through in your this.props. So in essence, you're right, the `param` can be named anything you like